.env.example

```diff
@@ -1,6 +1,6 @@
 export idp_domain=localhost:3000
 export idp_url=http://localhost:3000
-export irs_private_key_path=./config/irs_pk.key
+export irs_private_key_path=./config/demo_irs_pk.key
```

Comment:
I think we'd be better off making the env variable the entire key, instead of a path to the key. this means we have to keep the private keys in version control which is not idea. Even if they're just demo keys, it gives the appearance of poor secret management